base/libgit2/libgit2.jl

```diff
@@ -219,22 +219,22 @@ function branch!(repo::GitRepo, branch_name::AbstractString,
                  force::Bool=false,           # force branch creation
                  set_head::Bool=true)         # set as head reference on exit
     # try to lookup branch first
-    branch_ref = force ? nothing : lookup_branch(repo, branch_name)
-    if branch_ref === nothing
-        branch_rmt_ref = isempty(track) ? nothing : lookup_branch(repo, "$track/$branch_name", true)
+    branch_ref = Nullable{GitReference}(force ? nothing : lookup_branch(repo, branch_name))
```

Comment:
if it returns a not-found Nullable, does this end up wrapping it in two layers of Nullable or one?